### ISSUE TITLE
connection refused

### ISSUE THREAD

**Issue by Sarenrenegade** [hdus/SRTM-Downloader#22]
Hi,
I'm having trouble downloading some dataset. The usual pop-up window with the credential does not appear when I hit the download button. Is there a problem with the network? 
I have included a screenshot:
![error_srtm](https://user-images.githubusercontent.com/68898650/151198561-904c9a59-9479-4a90-bf2d-57aadefda813.PNG)

I'm using QGIS 3.10.16 and I have tried with another computer (same problem occured). I also checked my credentials at the eathdata website. Thanks in advance for your help.

**Comment by daniel7374859** [comment]
Hi @Sarenrenegade, had the same problem just now and tried about everything. It worked after I updated the software to latest version. Hope this helps!
best,
dan

**Comment by Skytled** [comment]
I'm experiencing the exact same issue with QGIS 3.22, 3.20.2 and 3.16.10. I'm getting "Connection refused" on every version of QGIS! Maybe it's a server issue?

**Comment by Sarenrenegade** [comment]
Hi ! I think it's a server issue, because the problem disapeared whithout
any reason... The only "workaround" is to directly download the tiles from
the Website or with another client (like GMT for instance).
S.

**Comment by TOUPaulToews** [comment]
Same issue for me today. Yesterday it has a timeout message, this morning it is connection refused.

**Comment by OpenSearcher** [comment]
I have the same issue WITH QGIS 3.24.2

**Comment by OlguinCarvajal** [comment]
I have the same problem. Has anyone been able to solve it?

**Comment by Sarenrenegade** [comment]
Yes and no. The service should be back in à few days (that's what happened
before). If you are in a hurry, try downloading it from GMT server or from
the nasa Website.
Good luck

Le mer. 29 juin 2022 à 17:04, OlguinCarvajal ***@***.***> a
écrit :

**Comment by hmehta111** [comment]
I am getting this error with QGIS 3.28.0

![image](https://user-images.githubusercontent.com/117301388/199545875-e874b1f5-6802-4117-9a8d-3a747de13de5.png)

**Comment by Sarenrenegade** [comment]
Yes me too, the NASA servers were disconnected. Things should be back to
normal.
Regards

**Comment by Anihall541** [comment]
Was anyone ever able to figure out how to address this? I'm uninstalled and reinstall the plugin, and still no luck. I'm also not seeing a solution to fix the problem on here, even though it's a closed thred
